kolibri/plugins/learn/assets/test/views/topics-page.spec.js

```diff
@@ -88,7 +88,7 @@ jest.mock('kolibri.resources');
 jest.mock('kolibri.urls');
 jest.mock('kolibri.coreVue.composables.useUser');
 jest.mock('../../src/composables/useContentLink');
-jest.mock('../../src/composables/useSearch');
+jest.mock('kolibri-common/composables/useBaseSearch');
```

Comment:
This seems to be throwing a module not found error in the tests.

Reply:
Fixed!